pyam/ixmp4.py

```diff
@@ -70,10 +74,21 @@ def write_to_ixmp4(platform: ixmp4.Platform | str, df):
                 f"{dimension}."
             )
 
+    # The "version" meta-indicator, added when reading from an ixmp4 platform,
+    # should not be written to the platform
+    if "version" in df.meta.columns:
+        logger.warning(
+            "The `meta.version` column was dropped when writing to the ixmp4 platform."
+        )
+        meta = df.meta.drop(columns="version")
```

Comment:
Could this result in an empty data frame except for the model and scenario index?

Reply:
yes, added a test-variant for this case